### PR TITLE
Refactor Mesh raycast

### PR DESCRIPTION
**Pull request by patak-cat**
Clean up after fixing: https://github.com/mrdoob/three.js/issues/7318

### CODE REVIEW

File: src/objects/Mesh.js [hunk 1]
```diff
@@ -145,128 +211,66 @@ THREE.Mesh.prototype.raycast = ( function () {
 					b = indices[ i + 1 ];
 					c = indices[ i + 2 ];
 
-					vA.fromArray( positions, a * 3 );
-					vB.fromArray( positions, b * 3 );
-					vC.fromArray( positions, c * 3 );
+					intersection = checkBufferGeometryIntersection( this, raycaster, ray,
```
Comment (mrdoob) [inline src/objects/Mesh.js]: Do you mind removing the line break here?

Comment (matias-capeletto) [inline src/objects/Mesh.js]: Ok, no problem

**Comment by matias-capeletto** [comment]
Line breaks have been removed from the branch

**Comment by mrdoob** [comment]
Many thanks!

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#7320

**Pull request merged**